src/App.tsx

```diff
@@ -1,9 +1,20 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const [query, setQuery] = useState('');
+  const visibleMovies = moviesFromServer.filter(movie => {
+    const title = movie.title.toUpperCase();
+    const description = movie.description.toUpperCase();
+    const queryInUpperCase = query.trim().toUpperCase();
+
+
+    return description.includes(queryInUpperCase)
```

Comment:
it would be nice to create separate function for lowerCasing and `includes` check